samples/ingestion/ingestion-client/FetchTranscription/TextAnalyticsProvider.cs

```diff
@@ -0,0 +1,274 @@
+// <copyright file="TextAnalyticsProvider.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace FetchTranscriptionFunction
```

Comment:
the namespace is about FetchTranscription, and the class is TextAnalytics, should be in a separate namespace?